bio/picard/mergesamfiles/wrapper.py

```
@@ -13,14 +13,24 @@
 extra = snakemake.params.get("extra", "")
 java_opts = get_java_opts(snakemake)
 
+if snakemake.threads == 1:
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)